lib/realtime/api/tenant.ex

```diff
@@ -74,7 +74,8 @@ defmodule Realtime.Api.Tenant do
       :max_channels_per_client,
       :max_joins_per_second,
       :suspend,
-      :notify_private_alpha
+      :notify_private_alpha,
+      :private_only
```

Comment:
double checking if a migration is not needed for this

Reply:
migrations was in a different PR and has defaults to false